src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -12,7 +12,7 @@ import useLocalize from '@hooks/useLocalize';
 import useNetwork from '@hooks/useNetwork';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
-import {openDraftWorkspaceRequest} from '@libs/actions/Policy';
+import {openDraftWorkspaceRequest} from '@libs/actions/Policy/Policy';
```

Comment:
Commenting on my own code here but I _think_ we should change all of these `@libs/actions` to `@userActions` - what do you think, reviewer?

I asked [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1715875342291149) but didn't get any responses.

Reply:
Yeah that change should be fine!

Reply:
Why do you think we should change this?

Reply:
just consistency, really - if we have the more specific shortcut I feel like we should use it?

Reply:
Yea, that makes sense. Let's do it!